tests/by-util/test_watch.rs

```diff
@@ -14,6 +14,12 @@ const TRUE_CMD: &str = "%SYSTEMROOT%\\System32\\rundll32.exe";
 #[cfg(not(windows))]
 const TRUE_CMD: &str = "true";
 
+#[cfg(windows)]
+const ECHO_HELLO_CMD: &str = "echo | set /p dummyName=hello";
+
+#[cfg(not(windows))]
+const ECHO_HELLO_CMD: &str = "echo -n hello";
```

Comment:
Maybe it helps to use `"printf hello"` instead?

Reply:
Thanks for the suggestion! I changed the command to use `printf`.